src/lambda/vue/DONOTUSE.vue

```diff
@@ -7,18 +7,19 @@
 
```

Comment:
Can we .gitignore this file and delete it from the repo?

Reply:
deleted